tests/generic_config_updater/generic_updater_test.py

```diff
@@ -0,0 +1,1059 @@
+import json
+import jsonpatch
+import os
+import shutil
+import unittest
+from imp import load_source
+from unittest.mock import Mock, call
+load_source('gu', \
+    os.path.join(os.path.dirname(__file__), '../..', 'generic_config_updater', 'generic_updater.py'))
```

Comment:
>generic_config_updater [](http://example.com/codeflow?start=54&length=22)

It's a module, we'd better import a module instead of loading a single python file